### ISSUE TITLE
error on import gltf 2.0 file

### ISSUE THREAD

**Issue by iwanPlays** [ksons/gltf-blender-importer#8]
When trying to import https://sketchfab.com/models/b4bdfa9b1d1d4506a41fee8c9b1ae728#

```
Traceback (most recent call last):
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 282, in execute
    [self.create_group(nodes[idx], None) for idx in scene['nodes']]
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 282, in <listcomp>
    [self.create_group(nodes[idx], None) for idx in scene['nodes']]
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 261, in create_group
    self.create_group(self.root['nodes'][idx], ob)
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 249, in create_group
    ob = self.create_mesh(node, self.root['meshes'][node['mesh']])
  File "C:\Users\i\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\io_scene_gltf\__init__.py", line 213, in create_mesh
    me = bpy.data.meshes.new(mesh['name'])
KeyError: 'name'

location: <unknown location>:-1


```

**Comment by Sphinxxxx** [comment]
Temporary fix for this one: I opened the .gltf file in a text editor and added random `name`s to all the `meshes` before importing it:

    ...
    "meshes": [
        {
            "name": "m01",
            "primitives": [
                ...
            ]
        },
        {
            "name": "m02",
            "primitives": [
                ...
            ]
        }
    ],
    ...

I still get this problem though: #9

**Comment by ksons** [comment]
This should be fixed thanks to @scurest #1 PR. Can you confirm?

**Comment by ksons** [comment]
![rendering](https://user-images.githubusercontent.com/482123/37737966-bfdcb33e-2d55-11e8-8470-b30406750737.png)

This is the scene above imported from Sketchfab and rendered with Cycles and Freestyle.
